### ISSUE TITLE
[Accessibility] The <img> Element does not have an alt attribute/alternate text

### ISSUE THREAD

**Issue by plabanTest1** [gonfunko/scratch-blocks#146]
Accessibility tools used: Lighthouse & axe DevTools

Ensure <img> elements have alternate text or a role of none or presentation

As per the tool, there are 29 elements that needs to be addressed . Few of the elements options are as below:
- menu-bar_hoverable_c6WFB.menu-bar_menu-bar-item_oLDa-:nth-child(2) > img:nth-child(1)
- menu-bar_help-icon_3Xtrt
- #react-tabs-0 > img[draggable="false"]
- gui_extension-button-icon_S4gDk
- sprite-selector-item_sprite-image_2QWuK

**Comment by gonfunko** [comment]
These are all part of Scratch's UI (not the blocks editor) and out of scope for this effort.
